app/pages/admin/index.tsx

```diff
@@ -32,14 +33,15 @@ class Admin extends Component<Props> {
     const {
       query: {session}
     } = this.props;
+    const feedbackUrl = getConfig()?.publicRuntimeConfig.FEEDBACK_SITE_URL;
     return (
       <DefaultLayout session={session} title="Administrator Dashboard">
         <div>
           <Row>
+            <ProgramAdministration />
+            <ReportingOperations />
             <UserManagement />
-            <ApplicationManagement />
-            <ProgramDataManagement />
-            <DevOpsManagement />
+            <ReportAProblem serviceUrl={feedbackUrl || '#'} />
```

Comment:
Eventually, we should either:
- not render `ReportAProblem` if there is no `feedbackUrl`
- require that `feedbackUrl` is defined

Reply:
I originally implemented this according to your first point, but immediately realized that makes the feature impossible to see and test until it's in production. We generally want things to be visible in dev and test if they are visible in prod, so we can verify.

I could move the `|| '#'` part up to the `feedbackUrl` assignment, but that seems basically equivalent.

Reply:
This is okay as-is. There's a code smell here given that in prod, if we forget to set the environment variable, then the application will deploy without any issue, but this link won't work. Ironically this is the link that should be used to report things that don't work

Reply:
Yes. I guess the question is, how do you require in the yamls that a given value be defined?

Reply:
https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions

https://github.com/bcgov/cas-ciip-portal/blob/9b5ac0a772f67a48c6b30c1da94d513a29abd9ab/helm/cas-ciip-portal/templates/deploy.yaml#L63 would become something like

```yaml
value: {{ required "A valid .Values.feedbackServiceUrl entry is required!" .Values.helpDeskServiceUrl }}
```

Reply:
Right - for some reason I got confused for a minute and thought we were talking about the Metabase URL. We can't make the fider URL required, because it only exists for production.

Therefore the existing solution correctly allows us to review the feature in dev/test using an `href="#"`, and fills in the correct URL for production.

Reply:
Well, it doesn't even exist for production anymore, given that Fider was not migrated to the silver cluster...
This would be replaced by the shared inbox email I suppose